tools/openapi2jsonschema/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7-slim
+
+RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
+
+RUN pip install git+https://github.com/avida/openapi2jsonschema.git
```

Comment:
Perhaps it is worth specifying the tag or the version of the library that we install from GitHub in order to eliminate errors when updating the version of this library?

Reply:
This library has no releases, its my fork of openapi2jsonschema tool which is abandoned atm (last merged PR was about 1.5 years ago). Will create v0.1 tag